<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::PreferenceDialog</class>
 <widget class="QDialog" name="ns::PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <widget class="QLabel" name="promptLabel">
     <property name="text">
      <string>Please fill the preference informations:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formPanel" native="true">
     <layout class="QFormLayout" name="formPanelLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>&amp;Path</string>
        </property>
        <property name="buddy">
         <cstring>pathEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="pathEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="valueLabel">
        <property name="text">
         <string>&amp;Value</string>
        </property>
        <property name="buddy">
         <cstring>valueEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="valueEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="opLayout1">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="getButton">
       <property name="text">
        <string>&amp;Get</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setButton">
       <property name="text">
        <string>&amp;Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setFontButton">
       <property name="text">
        <string>Set &amp;Font</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setColorButton">
       <property name="text">
        <string>Set Co&amp;lor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="opLayout2">
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetAllButton">
       <property name="text">
        <string>Reset &amp;All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>valueEdit</tabstop>
  <tabstop>getButton</tabstop>
  <tabstop>setButton</tabstop>
  <tabstop>setFontButton</tabstop>
  <tabstop>setColorButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>resetAllButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ns::PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ns::PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
